nixos/modules/services/networking/xrdp.nix

```diff
@@ -90,89 +99,110 @@ in
         '';
       };
 
-      confDir = mkOption {
-        type = types.path;
-        default = confDir;
-        defaultText = literalMD "generated from configuration";
-        description = lib.mdDoc "The location of the config files for xrdp.";
+      extraConfDirCommands = mkOption {
+        type = types.str;
+        default = "";
+        description = lib.mdDoc ''
+          Extra commands to run on the default confDir derivation.
+
+          New files need to be also added to `confDirExposeFiles` to appear in /etc/xrdp.
```

Comment:
`confDirExposeFiles` does not exist anymore.

Reply:
Because of your glob tip it became irrelevant.

Reply:
Yes, this is what I meant: the description still cites this option as like it exist.

Reply:
BTW I think @chvp will want to take a look on this before merge. I am not sure about the potential breakages.